<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InvoiceForm</class>
 <widget class="QWidget" name="InvoiceForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>InvoiceFormular</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../PiTres.qrc">
    <normaloff>:/resources/application_logo.svg</normaloff>:/resources/application_logo.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="minimumSize">
      <size>
       <width>326</width>
       <height>231</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="members">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string>Invoice by Members</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Enter Member-ID:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="memberUid">
         <property name="placeholderText">
          <string>Insert Member UID and press Enter</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QListWidget" name="memberUidList">
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="checkPrint">
           <property name="text">
            <string>Print Reminder (Don't send by EMail)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnUidInvoice">
           <property name="text">
            <string>Send Invoices</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="by_date">
      <attribute name="title">
       <string>Invoice by Date</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="5" column="2">
        <widget class="QPushButton" name="btnInvoiceAll">
         <property name="text">
          <string>Invoice all Members</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="btnInvoiceUntil">
         <property name="text">
          <string>Invoice Members</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="btnInvoiceNew">
         <property name="text">
          <string>Invoice new Members</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>New Member since</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDateEdit" name="newMemberDate">
         <property name="displayFormat">
          <string>ddd dd. MMMM yyyy</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Member-Until Date</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDateEdit" name="memberUntil">
         <property name="displayFormat">
          <string>ddd dd. MMMM yyyy</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="checkPrintDate">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Print Reminder (Don't send by EMail)</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../PiTres.qrc"/>
 </resources>
 <connections/>
</ui>
